default-operator-config.yaml

```diff
@@ -1,21 +1,653 @@
+    v1.1.0:
+    - v1.2.0
+    - v1.3.0
+    - v1.4.0
+    - v1.5.0
+    - v1.6.0
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.2.0:
+    - v1.3.0
+    - v1.4.0
+    - v1.5.0
+    - v1.6.0
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.3.0:
+    - v1.4.0
+    - v1.5.0
+    - v1.6.0
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.4.0:
+    - v1.5.0
+    - v1.6.0
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.5.0:
+    - v1.6.0
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.6.0:
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.7.0:
+    - v1.7.1
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.7.1:
+    - v1.8.0
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.8.0:
+    - v1.9.0
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.9.0:
+    - v1.10.0
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.10.0:
+    - v1.11.0
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.11.0:
+    - v1.12.0
+    - v1.13.0
+    - v1.14.1
+    v1.12.0:
+    - v1.13.0
+    - v1.14.1
+    v1.13.0:
+    - v1.14.1
+    v1.14.0:
+    - v1.14.1
+  metadata:
+    datastore: cockroachdb
+  name: cockroachdb
+  nodes:
+  - id: v1.14.1
+    migration: add-caveats
+    tag: v1.14.1
+  - id: v1.14.0
+    migration: add-caveats
+    tag: v1.14.0
+  - id: v1.13.0
+    migration: add-metadata-and-counters
+    tag: v1.13.0
+  - id: v1.12.0
+    migration: add-metadata-and-counters
+    tag: v1.12.0
+  - id: v1.11.0
+    migration: add-metadata-and-counters
+    tag: v1.11.0
+  - id: v1.10.0
+    migration: add-metadata-and-counters
+    tag: v1.10.0
+  - id: v1.9.0
+    migration: add-metadata-and-counters
+    tag: v1.9.0
+  - id: v1.8.0
+    migration: add-metadata-and-counters
+    tag: v1.8.0
+  - id: v1.7.1
+    migration: add-metadata-and-counters
+    tag: v1.7.1
+  - id: v1.7.0
+    migration: add-metadata-and-counters
+    tag: v1.7.0
+  - id: v1.6.0
+    migration: add-metadata-and-counters
+    tag: v1.6.0
+  - id: v1.5.0
+    migration: add-transactions-table
+    tag: v1.5.0
+  - id: v1.4.0
+    migration: add-transactions-table
+    tag: v1.4.0
+  - id: v1.3.0
+    migration: add-transactions-table
+    tag: v1.3.0
+  - id: v1.2.0
+    migration: add-transactions-table
+    tag: v1.2.0
+  - id: v1.1.0
+    migration: add-transactions-table
+    tag: v1.1.0
+  - id: v1.0.0
+    migration: add-transactions-table
+    tag: v1.0.0
+- edges:
+    v1.7.0:
```

Comment:
Why does CRDB only have edges starting at v1.7.0?

Reply:
these edges are for mysql, which we introduced in 1.7.0